### PR TITLE
COM-417: fix select

### PR DESCRIPTION
**Pull request by TheEnmanuel23**
## Please delete checklist items and sections that are not relevant

**Checklist**

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

**Please check if the PR fulfills these requirements**

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Storybook or docs have been added / updated (for bug fixes / features)

**What is the ticket / issue associated with this change?** (Link to the ticket or issue)
https://impulsumvc.atlassian.net/browse/COM-417
**Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

**Other information**:
I fixed the border radius, as well, from 0.4rem to 0.375rem. I took the opportunity to remove the import of the inputStyle object, since after fixing the mentioned issues, only 2 properties were being used from that object and that we could perfectly set them in the field object of the baseStyles
**Storybook Link** (Link to a local storybook url to review new functionality)
https://5eed577398d08d0022b52d76-xqhmgoosxi.chromatic.com/?path=/story/components-primitives-select--all

### CODE REVIEW

**Comment by TheEnmanuel23** [comment]
after modifying the width and height to 20px in the div or container of the icon, and using 24px (1.5rem) as padding right, they look like this:
image

**Comment by TheEnmanuel23** [comment]
I already fixed it, I ended up fixing the padding right: 
- For small and Medium sizes:
20px (icon width) + 12px between input and icon + 12px between icon and border = `1.25rem+ 0.75rem + 0.75rem = 2.75rem` as padding right and I fixed the icon position in order to match it with the design
image

- For large sizes:
20px (icon width) + 12px between input and icon + 16px between icon and border = `1.25rem+ 0.75rem + 1rem = 3rem` as padding right and I fixed the icon position in order to match it with the design
image

after those fixes the implementation matches exactly with the design

File: packages/components/src/theme/ComposerTheme/Components/Select.ts [hunk 1]
```diff
@@ -37,6 +53,10 @@ const baseStyle: PartsStyleFunction<typeof parts> = ({ theme, iconSize }) => {
       boxShadow: 'none',
       color: 'text-primary',
       placeholderColor: 'text-secondary',
+      minW: '8rem',
```
Comment (tkanzakic) [inline packages/components/src/theme/ComposerTheme/Components/Select.ts:56]: IMO we should not do this, this is up to the user of the library

**Review by tkanzakic — COMMENTED** [review]
It would be great to update the storybook to match the Input component

**Review by tkanzakic — APPROVED** [review]

**Review by guilean — APPROVED** [review]

**Pull request merged**